community-connectors/google-fit/README.md

```diff
@@ -0,0 +1,50 @@
+# Google Fit Community Connector for Data Studio
+
+![Screenshot](./screenshot.jpg?raw=true "Screenshot")
+
+*This is not an official Google product*
+
+This [Data Studio](https://datastudio.google.com) [Community
+Connector](https://developers.google.com/datastudio/connector) allows users to query
+[Google Fit](https://fit.google.com) for fitness data.
+This connector uses the Google Fit [REST API](https://developers.google.com/fit/rest/).
+
+## Supported data
+
+The Google Fit Community Connector supports the following types of Google Fit data.
+
+- Activity (Walking, Running, Calisthenics, etc.)
+- Steps
+- Weight
+
+## Try out the connector
+
+You can use the latest version of the Google Fit Community Connector using this link: [Google Fit Community Connector](https://datastudio.google.com/datasources/create?connectorId=AKfycbxlNBvs3B1qMjPdXJzhmg0WrtPw_rakGnRdC2ImgAMUx-FX60HXHut0)
```

Comment:
When I try this link, I get an error saying "The connector manifest could not be retrieved or is invalid. Check the connector and try again." Could you double check that this is deployed an accessible through this link?

Reply:
I'm not sure why it's not working. I guess the error message is referring to the appsscript.json?

I'm pretty sure I tested this before and it worked. Though I get the same message when I access the url I have a dashboard that's currently working where I installed the connector from the same url. The connector ID is correct, and the files are saved in the project as they are committed here.

Reply:
Ok. it seems I hadn't shared the Apps Script project and made it public. It should work now I think.